app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container mt-5">
+  <div class="row justify-content-center">
+    <div class="col-md-6">
+      <div class="card">
+        <div class="card-header"><h2 class="mb-0">Log in</h2></div>
+        <div class="card-body">
+          <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+            <div class="form-group">
+              <%= f.label :email %>
+              <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
```

Comment:
- [ ] an error occurred running this app about an undefined email, kindly fix it
![image](https://github.com/Jawad-Nazari/Recipe-app/assets/66097371/e0c59b8a-1dd9-4dc8-9291-b197d26007ad)

Reply:
Hi @VICTOR-CHUKWU , 
Thanks for your time and for reviewing our code. We will come up with a solution. 🙏

Reply:
- As pointed out by the previous reviewer, I still get the same error when running your app. Kindly fix it.



![Screenshot from 2023-08-12 14-25-38](https://github.com/Jawad-Nazari/Recipe-app/assets/59397159/953af804-3081-414d-9b22-35c588537fe1)